config/config.go

```diff
@@ -48,6 +48,11 @@ func Load() (*Shield, error) {
 		options = append(options, config.WithPath(filepath.Join(currentHomeDir, ".config")))
 	}
 
+	// override all config sources and prioritize one from file
+	if configfileFromFlag != "" {
```

Comment:
@mabdh This is for server configs, right?

Reply:
Yes it is for server, maybe naming this explicitly with `server` will give more clarity, will update this thanks @ravisuhag